CONTRIBUTING.md

```diff
@@ -651,3 +651,79 @@ Names of files and directories should be in lowercase, with dashes between words
+
+### How can I get a committer to look at my PR?
+
+- Simply waiting. Reviewers frequently browse open PRs and may happen to run across yours and take a look.
+- Directly mentioning someone (`@committergithubhandlehere`) and asking them nicely
+- Posting in one of the channels made for this purpose
+  - The current "PRs ready for review" or "PRs already reviewed" threads in the [NixOS Discourse](https://discourse.nixos.org/c/dev/14) (of course choose the one that applies to your situation)
+  - The [Nixpkgs Review Requests Matrix room](https://matrix.to/#/#review-requests:nixos.org).
+
+### CI failed or got stuck on my PR, what do I do?
+
+First ensure that the failure is actually related to your change. Sometimes, the CI system simply has a hiccup or the check was broken by someone else before you made your changes. Read through the error message; it's usually quite easy to tell whether it is caused by anything you did by checking whether it mentions the component you touched anywhere. If it is indeed caused by your change, obviously try to fix it. Your PR is unlikely to be merged if it has a known issue and it is the purpose of CI to alert you aswell as reviewers to these issues.
+
+ofBorg builds can often get stuck, particularly in PRs targeting `staging` and in builders for the Darwin platform. Reviewers will know how to handle them or when to ignore them. Don't worry about it.
+If there is a build failure however and it happened due to a package related to your change, you need to investigate it of course. If ofBorg reveals the build to be broken on some platform and you don't have access to that platform, you should set your package's `meta.broken` accordingly.
+
+When in any doubt, please simply ask via a comment in your PR or through one of the help channels.
+
+### A reviewer requested a bunch of insubstantial changes on my PR
+
+The people involved in Nixpkgs care about code quality because, once in Nixpkgs, it needs to be maintained for many years to come. It is therefore likely that other people will ask you to do some things in another way or adhere to some standard.
+Sometimes however, they also care a bit too much and may ask you to adhere to a personal preference of theirs. It's not always easy to tell which is which and whether the requests are critically important to merging the PR.
+Sometimes another reviewer may also come along with totally different opinions on some points too.
+
+It is convention to mark review comments that are not critical to the PR as nitpicks but this is not always followed. As the PR author, you should still take a look at these as they will often reveal best practices and unwritten rules that usually have good reasons behind them and you may want to incorporate them into your modus operandi.
+
+Please keep in mind that reviewers are almost always well meaning here. Their intent is not to denounce your code, they simply want your code to be as good as it can be.
```

Comment:
```suggestion
Please keep in mind that reviewers are almost always mean well. Their intent is not to denounce your code, they simply want your code to be as good as it can be.
```